### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by LinuxDonald**
If anyone have corrections for me I´m to avaliable on the Discord XoM :)

### CODE REVIEW

**Review by CovertDuck — APPROVED** [review]

**Comment by LinuxDonald** [comment]
At the weekend I will upload more changes. An review again would be nice

File: XIV on Mac/de.lproj/Main.strings [hunk 1]
```diff
@@ -117,16 +117,16 @@
 "9gQ-FA-62f.title" = "D2013.06.18.0000.0000";
 
 /* Class = "NSMenuItem"; title = "Substitutions"; ObjectID = "9ic-FL-obx"; */
-"9ic-FL-obx.title" = "Substitutions";
+"9ic-FL-obx.title" = "Ersetzungen";
 
 /* Class = "NSButtonCell"; title = "Quit XIV on Mac upon game exit"; ObjectID = "9ie-LK-Dur"; */
-"9ie-LK-Dur.title" = "Quit XIV on Mac upon game exit";
+"9ie-LK-Dur.title" = "XIV on Mac beim Beenden des Spiels beenden";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Main.strings:123]: Maybe use

> *nach Beenden des Spiels schließen*

Your translation is OK but it uses word repetition again (Beenden ... beenden)

**Review by Shirk — CHANGES_REQUESTED** [review]
I added comments and notes pointing out a number of issues that should be addressed:
- there's a number of singular / plural issues
- various messages can and should be simplified to use less winded wording / complex nesting (suggestions and corrections provided in my comments)
- certain phrases like "Ordner" have become highly uncommon for use in a PC environment these days

File: XIV on Mac/de.lproj/Main.strings [hunk 2]
```diff
@@ -24,76 +24,76 @@
...
-"6Z2-gP-gHf.title" = "Cancel";
+"6Z2-gP-gHf.title" = "Abbrechen";
 
 /* Class = "NSMenuItem"; title = "Redo"; ObjectID = "6dh-zS-Vam"; */
-"6dh-zS-Vam.title" = "Redo";
+"6dh-zS-Vam.title" = "Wiederholen";
 
 /* Class = "NSTextFieldCell"; title = "HUD Options are informational items displayed in the upper left corner during play that can aid you in determining how in-game settings are affecting your performance. They are entirely optional."; ObjectID = "74A-cF-fLc"; */
-"74A-cF-fLc.title" = "HUD Options are informational items displayed in the upper left corner during play that can aid you in determining how in-game settings are affecting your performance. They are entirely optional.";
+"74A-cF-fLc.title" = "HUD Optionen sind Informationselemente, die während des Spiels in der oberen linken Ecke angezeigt werden und dir dabei helfen können, herauszufinden, wie sich die Einstellungen im Spiel auf deine Leistung auswirken. Sie sind völlig optional.";
```
[the remark this answers is not in the document]
Comment (LinuxDonald) [reply XIV on Mac/de.lproj/Main.strings:81]: Have you maybe an Screenshot how it looks in game? I would then add the fix for the English version too. I don't have at the Moment an active account.

[the remark this answers is not in the document]
Comment (Shirk) [reply XIV on Mac/de.lproj/Main.strings:81]: I need to correct my statement a little bit - there's 3 locations for the HUD(s) if you tick all the boxes:
Screenshot 2023-07-09 at 11 50 11@2x

Top-Left shows all the advanced DXVK options, Bottom-Left shows the DXVK status and to right is the Metal-HUD

[the remark this answers is not in the document]
Comment (LinuxDonald) [reply XIV on Mac/de.lproj/Main.strings:81]: thx

File: XIV on Mac/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -109,26 +109,26 @@ Aktiviere dies nicht auf Nicht-Retina-Displays, es sei denn, du benötigen Scree
...
 "INSTALLER_FOUND_MESSAGE" = "FINAL FANTASY XIV ONLINE Installation gefunden!";
 "INSTALLER_FOUND_INFORMATIVE" = "Möchtest du die Installation unter %@ verwenden?";
 "INSTALLER_NOT_DETECTED_MESSAGE" = "Es wurden keine FINAL FANTASY XIV ONLINE-Installationen gefunden..";
-"INSTALLER_NOT_DETECTED_INFORMATIVE" = "Möchtest du einen Ordner manuell auswählen?";
+"INSTALLER_NOT_DETECTED_INFORMATIVE" = "Möchtest du ein Verzeichnis manuell auswählen?";
 
-"INSTALLER_PATH_TITLE" = "Wähle den Ordner mit der bestehenden Installation";
-"INSTALLER_PATH_SUBTITLE" = "Sie sollte die Ordner \"game\" und \"boot\" sowie die ausführbare Datei des Spiels enthalten.";
+"INSTALLER_PATH_TITLE" = "Wähle das existierende Installationsverzeichnis aus";
+"INSTALLER_PATH_SUBTITLE" = "Er sollte die Verzeichnisse \"game\" und \"boot\" sowie die ausführbare Datei des Spiels enthalten.";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Localizable.strings:123]: > *Es* sollte

File: XIV on Mac/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -65,41 +65,41 @@
...
 "FIRSTAID_CFGCHECK_HBAO_EXP" = "Die HBAO-Stufen von SSAO (Screen Space Ambient Occlusion) werden auf dem Mac nicht unterstützt und können dazu führen, dass Grafiken mit schwarzen Löchern dargestellt werden.";
 "FIRSTAID_CFGCHECK_TESSELATION_TITLE" = "Tesselation";
-"FIRSTAID_CFGCHECK_TESSELATION_EXP" = "Werte über \"Normal\" sind dafür bekannt, dass sie auf Apple Silicon Macs zu erheblichen Leistungsproblemen führen und keine visuelle Verbesserung bieten.";
+"FIRSTAID_CFGCHECK_TESSELATION_EXP" = "Werte über \"Normal\" sind dafür bekannt, dass sie auf Apple Silicon Macs zu erheblichen Leistungsproblemen führen und keine optische Verbesserung bieten.";
 
 "SELECT_GAME_PATH_PANEL_TITLE" = "Wähle den FFXIV Installationsort";
-"SELECT_GAME_PATH_PANEL_SUBTITLE" = "Sie sollte die Ordner \"game\" und \"boot\" enthalten.";
-"SELECT_GAME_PATH_ERROR_MESSAGE" = "Der Ordner, den du für die Installation deines Spiels gewählt hast, scheint der falsche zu sein";
-"SELECT_GAME_PATH_ERROR_INFORMATIVE" = "Möchtest du es immer noch benutzen?";
+"SELECT_GAME_PATH_PANEL_SUBTITLE" = "Er sollte die Ordner \"game\" und \"boot\" enthalten.";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Localizable.strings:89]: > *Es* sollte

File: XIV on Mac/de.lproj/Localizable.strings [hunk 5]
```diff
@@ -109,26 +109,26 @@ Aktiviere dies nicht auf Nicht-Retina-Displays, es sei denn, du benötigen Scree
...
-"INSTALLER_NOT_DETECTED_INFORMATIVE" = "Möchtest du einen Ordner manuell auswählen?";
+"INSTALLER_NOT_DETECTED_INFORMATIVE" = "Möchtest du ein Verzeichnis manuell auswählen?";
 
-"INSTALLER_PATH_TITLE" = "Wähle den Ordner mit der bestehenden Installation";
-"INSTALLER_PATH_SUBTITLE" = "Sie sollte die Ordner \"game\" und \"boot\" sowie die ausführbare Datei des Spiels enthalten.";
+"INSTALLER_PATH_TITLE" = "Wähle das existierende Installationsverzeichnis aus";
+"INSTALLER_PATH_SUBTITLE" = "Er sollte die Verzeichnisse \"game\" und \"boot\" sowie die ausführbare Datei des Spiels enthalten.";
 "INSTALLER_PATH_INVALID_MESSAGE" = "Ungültiges FFXIV-Verzeichnis";
-"INSTALLER_PATH_INVALID_INFORMATIVE" = "Sie sollte die Ordner \"game\" und \"boot\" sowie die ausführbare Datei des Spiels enthalten.";
+"INSTALLER_PATH_INVALID_INFORMATIVE" = "Er sollte die Verzeichnisse \"game\" und \"boot\" sowie die ausführbare Datei des Spiels enthalten.";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Localizable.strings:125]: > *Es* sollte

**Review by Shirk — COMMENTED** [review]
Two small er / es changes and you get my OK

**Comment by LinuxDonald** [comment]
Thank you but im still not done yet. There is still more to translate... I hope I get it done this week.

**Comment by Shirk** [comment]
> Thank you but im still not done yet. There is still more to translate... I hope I get it done this week.

Looking good now, lmk when you're done and I'll give it a review :)

**Comment by LinuxDonald** [comment]
@Shirk i´m done. I think there are some wrong translations....

**Comment by Shirk** [comment]
Looks pretty good, well done.
I can add some comments later tonight once I'm back at my Mac if you like. 👍️

**Comment by LinuxDonald** [comment]
Yeah that would be nice :) We want both an good translation for it :)

File: XIV on Mac/de.lproj/Main.strings [hunk 6]
```diff
@@ -381,97 +381,97 @@
...
-"YEy-JH-Tfz.title" = "Find and Replace…";
+"YEy-JH-Tfz.title" = "Suchen und Ersetzen...";
 
 /* Class = "NSButtonCell"; title = "Device Info"; ObjectID = "YOz-MM-jki"; */
-"YOz-MM-jki.title" = "Device Info";
+"YOz-MM-jki.title" = "Gerät Info";
 
 /* Class = "NSMenuItem"; title = "Start Speaking"; ObjectID = "Ynk-f8-cLZ"; */
-"Ynk-f8-cLZ.title" = "Start Speaking";
+"Ynk-f8-cLZ.title" = "Sprechen beginnen";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Main.strings:435]: Checked the default translation for macOS (which I'm guessing is what these keys are for) - this is apparently "Diktat starten"

File: XIV on Mac/de.lproj/Main.strings [hunk 7]
```diff
@@ -480,82 +480,82 @@
 "c2g-j3-NbA.title" = "Microsoft .NET 6.0";
 
 /* Class = "NSMenu"; title = "Transformations"; ObjectID = "c8a-y6-VQd"; */
-"c8a-y6-VQd.title" = "Transformations";
+"c8a-y6-VQd.title" = "Verwandlungen";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Main.strings:483]: Works, but could also be "Transformationen", which is the default system text on German macOS

File: XIV on Mac/de.lproj/Main.strings [hunk 8]
```diff
@@ -480,82 +480,82 @@
...
-"d5H-SM-vjt.title" = "XIV On Mac can work with Mac, Windows, and Steam licenses. You must already own the license type you choose, if you are unsure you can check your account status through Mogstation. Steam accounts must have been previously linked to your Final Fantasy XIV service account by logging in on a Windows PC, and further require the macOS version of Steam to be running.";
+"d5H-SM-vjt.title" = "XIV On Mac kann mit Mac-, Windows- und Steam-Lizenzen verwendet werden. Wenn du dir nicht sicher bist, kannst du den Status deines Kontos über Mogstation überprüfen. Steam-Konten müssen zuvor mit deinem Final Fantasy XIV-Dienstkonto verknüpft worden sein, indem du dich auf einem Windows-PC eingeloggt hast, und außerdem muss die macOS-Version von Steam laufen.";
 
 /* Class = "NSButtonCell"; title = "Troubleshooting"; ObjectID = "d7x-jG-QDv"; */
-"d7x-jG-QDv.title" = "Troubleshooting";
+"d7x-jG-QDv.title" = "Fehlersuche";
 
 /* Class = "NSMenuItem"; title = "Make Lower Case"; ObjectID = "d9M-CD-aMd"; */
-"d9M-CD-aMd.title" = "Make Lower Case";
+"d9M-CD-aMd.title" = "Kleinschreibung vornehmen";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Main.strings:504]: Works, standard translation would be "Kleinbuchstaben" (which is weird tbh.)

File: XIV on Mac/de.lproj/Main.strings [hunk 9]
```diff
@@ -633,112 +633,112 @@
...
-"piU-pX-vSl.title" = "If your game is freezing at startup, \"Compiling Shaders…\" forever, missing lighting or randomly freezing, the problem may be with your shader cache. Deleting it will cause a new one to be rebuilt, though it may result in occasional stutters during play initially.";
+"piU-pX-vSl.title" = "Wenn dein Spiel beim Start einfriert, \"Compiling Shaders...\" ewig dauert, die Beleuchtung fehlt oder das Spiel zufällig einfriert, liegt das Problem möglicherweise an deinem Shader-Cache. Wenn du ihn löschst, wird ein neuer Cache erstellt, auch wenn dies anfangs zu gelegentlichen Stottern beim Spielen führen kann.";
 
 /* Class = "NSMenuItem"; title = "Find Next"; ObjectID = "q09-fT-Sye"; */
-"q09-fT-Sye.title" = "Find Next";
+"q09-fT-Sye.title" = "Nächste finden";
 
 /* Class = "NSButtonCell"; title = "Launch Vanilla Game"; ObjectID = "qDd-CO-Fjs"; */
-"qDd-CO-Fjs.title" = "Launch Vanilla Game";
+"qDd-CO-Fjs.title" = "Vanille-Spiel starten";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Main.strings:666]: > "Unmodifiziertes Spiel starten" 
is probably what you want to use here, for use Vanille is mainly something we associate with ice cream :D

File: XIV on Mac/de.lproj/Main.strings [hunk 10]
```diff
@@ -633,112 +633,112 @@
...
-"sdf-Re-eaB.label" = "Done";
+"sdf-Re-eaB.label" = "Fertig";
 
 /* Class = "NSMenuItem"; title = "Edit Registry"; ObjectID = "snW-S8-Cw5"; */
-"snW-S8-Cw5.title" = "Edit Registry";
+"snW-S8-Cw5.title" = "Registry bearbeiten";
 
 /* Class = "NSMenuItem"; title = "Data Detectors"; ObjectID = "tRr-pd-1PS"; */
-"tRr-pd-1PS.title" = "Data Detectors";
+"tRr-pd-1PS.title" = "Daten-Detektive";
```
Comment (Shirk) [inline XIV on Mac/de.lproj/Main.strings:696]: Cross-checking with German macOS this item doesn't exist unless the language is set to English - weird.

**Review by Shirk — APPROVED** [review]
I could work with this translation as a native speaker.
I added a few extra comments but it gets my review.

**Comment by LinuxDonald** [comment]
@Shirk one last review than its done :) Thank you

**Review by Shirk — APPROVED** [review]

**Comment by LinuxDonald** [comment]
@marzent could you maybe merge it?

File: XIV on Mac/de.lproj/Main.strings [hunk 11]
```diff
@@ -480,82 +480,82 @@
...
-"dkA-rO-6VD.title" = "Graphics and Compute Pipelines";
+"dkA-rO-6VD.title" = "Grafik und Compute Pipelines";
 
 /* Class = "NSMenuItem"; title = "Select Game Path"; ObjectID = "dxd-Xu-JNa"; */
-"dxd-Xu-JNa.title" = "Select Game Path";
+"dxd-Xu-JNa.title" = "Spielpfad auswählen";
 
 /* Class = "NSButtonCell"; title = "Anamnesis"; ObjectID = "eFL-CT-TsU"; */
-"eFL-CT-TsU.title" = "Anamnesis";
+"eFL-CT-TsU.title" = "Anamnese";
```
Comment (marzent) [inline XIV on Mac/de.lproj/Main.strings:537]: Anamnesis is a name and shouldn't be translated

File: XIV on Mac/de.lproj/Main.strings (hunk 10 above)
Comment (marzent) [reply XIV on Mac/de.lproj/Main.strings:696]: I think "Datenerkennung" would be a better fit here

File: XIV on Mac/de.lproj/Main.strings [hunk 12]
```diff
@@ -633,112 +633,112 @@
...
-"snW-S8-Cw5.title" = "Edit Registry";
+"snW-S8-Cw5.title" = "Registry bearbeiten";
 
 /* Class = "NSMenuItem"; title = "Data Detectors"; ObjectID = "tRr-pd-1PS"; */
-"tRr-pd-1PS.title" = "Data Detectors";
+"tRr-pd-1PS.title" = "Daten-Detektive";
 
 /* Class = "NSButtonCell"; title = "Frame Time Graph"; ObjectID = "ti2-f2-LWe"; */
-"ti2-f2-LWe.title" = "Frame Time Graph";
+"ti2-f2-LWe.title" = "Rahmenzeitdiagramm";
```
Comment (marzent) [inline XIV on Mac/de.lproj/Main.strings:699]: I would suggest "Frametime Diagramm"; "Rahmenzeitdiagramm" is a literal translation.

**Comment by LinuxDonald** [comment]
Done

**Review by marzent — APPROVED** [review]

**Comment by Shirk** [comment]
@marzent - is there a reason why you have the Menu entries translated at all?
For one thing macOS should simply use the actual system texts if you don't have them and for another the "data detectors" don't exist on German systems.

**Comment by marzent** [comment]
All localisation strings were originally auto-created by Xcode, I assumed this is the most "correct" way of handling them. 
If the default menu entries don't need that and it will be handled by the OS I wouldn't mind removing them, if that is the case.

**Comment by LinuxDonald** [comment]
For me Main Menu is in English on a German MacOS system. I don't use my translations there yet.

**Comment by Shirk** [comment]
> For me Main Menu is in English on a German MacOS system. I don't use my translations there yet.

There's something very weird going on with the translations.. if I remove all the keys from the .strings file I get a mixed German and English result:
Screenshot 2023-07-20 at 20 38 06@2x

I'll take a note to do some research into this so maybe a future PR can remove the default entries.
Thank you both for making XIV-on-Mac more comfortable for those of us that prefer their UI in German!

**Pull request merged**